src/env.d.ts

```diff
@@ -6,7 +6,7 @@ declare const webpackEnv: {
     readonly target: 'Chromium' | 'Firefox' | 'WKWebview' | 'E2E' | undefined
     readonly firefoxVariant: 'android' | 'desktop' | 'GeckoView' | undefined
     readonly genericTarget: 'facebookApp' | 'browser'
-    readonly responsiveTarget: 'xs' | 'sm' | 'md' | 'lg' | 'xl'
+    readonly perferResponsiveTarget: 'xs' | 'sm' | 'md' | 'lg' | 'xl'
```

Comment:
I suggest only 2 value, `preferResponsiveTarget: 'desktop' | 'mobile'`

Reply:
Actually, we use a lots of CSS media queries like `[theme.breakpoints.down('xs')]]` in the codebase. I think it's easier to understand if we follow the same values in the JS even though we haven't used the values expected `xs`.

Reply:
If you're not using these values, please comment them out at this moment.